README.md

```diff
@@ -0,0 +1,37 @@
+# GOB-Airflow
+
+GOB Airflow orchestrates GOB data pipelines.
+
+# Infrastructure
+
+A running [GOB infrastructure](https://github.com/Amsterdam/GOB-Infra)
+is required to run this component.
+
+You might want to stop GOB-Workflow however to avoid conflicts with queue's handling.
+
+# Docker
+
+## Requirements
+
+* docker-compose >= 1.17
+* docker ce >= 18.03
+
+## Run
+
+```bash
+docker-compose build
+docker-compose up
+```
+
+# Airflow Admin GUI
+
+Go to [Airflow Admin GUI](http://localhost:8088/admin/) with your browser.
+
+In GUI you can:
+
+    1. Enable DAG(s).
```

Comment:
I do not know the Airflow GUI but probably add the menu items and what should be typed in to get the example working.